route/v1/types.go

```diff
@@ -282,3 +282,10 @@ const (
 	//          Note that this will not match acme.test only *.acme.test
 	WildcardPolicySubdomain WildcardPolicyType = "Subdomain"
 )
+
+// Route Annotations
+const (
+	// OpenShiftAllowNonDNSCompliantHostAnnotation is an instruction to allow non-DNS Compliant
+	// host names in a route configuration
+	OpenShiftAllowNonDNSCompliantHostAnnotation = "openshift.io/allow-non-dns-compliant-host"
```

Comment:
"OpenShift" seems redundant.
```suggestion
	AllowNonDNSCompliantHostAnnotation = "openshift.io/allow-non-dns-compliant-host"
```